elasticdl/python/common/tensor.py

```diff
@@ -6,10 +6,11 @@
     dtype_numpy_to_tensor,
     dtype_tensor_to_numpy,
 )
+from elasticdl.python.common.log_utils import default_logger as logger
```

Comment:
Probably moving class Tensor definition to the beginning of the file will make this file more readable.

Reply:
Done.